atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/command/TestTimestampCommand.java

```diff
@@ -152,6 +152,7 @@ public void testWithoutFile() throws Exception {
             String output = runner.run(true, false);
             try {
                 Scanner scanner = new Scanner(output);
+                scanner.nextLine(); // Skip deprecation warning
```

Comment:
Could instead check that the proper message is logged. Skipping is probably fine though